frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -2588,9 +2675,12 @@ void ProgramStructure::populateOutputNames() {
         "mark_to_drop",
         "hash",
         "resubmit",
+        "resubmit_preserving_field_list",
```

Comment:
I will double-check this with my own small tests, but is resubmit_preserving_field_list a new primitive internal to p4c only, or do these changes also require new primitives in the BMv2 JSON file that BMv2 reads?

Reply:
This is compiled to the standard recirculare bmv2 primitive

Reply:
Sounds good.  Thanks.